docs/built-in-rules.md

```diff
@@ -193,4 +193,46 @@ Azure supports two deployment models: Classic and Azure Resource Manager (ARM).
+
+**Recommendation**: Set the *properties.enabled* property to `false` to ensure that direct access to the management REST API is disabled.
+#### Severity: 3
+
+### TA-000034: API Management minimum API version should be set to 2019-12-01 or higher
+To prevent service secrets from being shared with read-only users, the minimum API version should be set to 2019-12-01 or higher.
+
+**Recommendation**: Set the *apiVersionContstraint.minAPIversion* property to 2019-12-01 or higher to prevent service secrets from being shared with read-only users.
+#### Severity: 2
+
+### TA-000035: API Management Named Values secrets should be stored in Azure Key Vault
+Named Values are a collection of name and value pairs in each API Management service. Secret values can be stored either as encrypted text in API Management (custom secrets) or by referencing secrets in Azure Key Vault. To improve security of API Management and secrets, reference secret Named Values from Azure Key Vault. Azure Key Vault supports granular access management and secret rotation policies.
+
+**Recommendation**: To utilize secrets stored in Key Vault for Azure API Management, learn more [here](https://learn.microsoft.com/azure/api-management/api-management-howto-properties?tabs=azure-portal#key-vault-secrets).
+#### Severity: 2
+
+### TA-000036: API Management services should use a virtual network
+Azure Virtual Network deployment provides enhanced security, isolation and allows you to place your API Management service in a non-internet routable network that you control access to. These networks can then be connected to your on-premises networks using various VPN technologies, which enables access to your backend services within the network and/or on-premises. The developer portal and API gateway, can be configured to be accessible either from the Internet or only within the virtual network.
+
+**Recommendation**: To learn more about virtual networks for Azure API Management, please visit [here](https://aka.ms/apim-vnet). To see an example of how to configure a VNet via Azure Policy, please visit [here](https://learn.microsoft.com/samples/azure/azure-quickstart-templates/api-management-create-with-internal-vnet-application-gateway/).
+#### Severity: 2
+
+### TA-000037: API Management subscriptions should not be scoped at the all API scope
+API Management subscriptions should be scoped to a product or an individual API instead of all APIs, which could result in an excessive data exposure.
+
+**Recommendation**: To ensure the API Management service is not configured at the all API scope, please set the *properties.scope* property to a value other than */apis*. For example, */product/{productId}*, or */apis/{apiId}*.
+#### Severity: 2
+
+### TA-000038: API Management calls to API backends should be authenticated
+Calls from API Management to backends should use some form of authentication, whether via certificates or credentials. Does not apply to Service Fabric backends.
```

Comment:
Do you have a test case for `Does not apply to Service Fabric backends.`? If not, can you add one please?

Reply:
Thanks Anais! I've verified with the APIM team on this, I've added a new check in the BuiltInRules for this case, and a new test case. I will also work with the APIM team to update the runtime policy to include this check as well.